backend/src/schemas/bets.py

```diff
@@ -41,4 +41,7 @@ class WagerCreateContext(BaseModel):
     amount: int
     yes: bool
     bet_uuid: str
-    
\ No newline at end of file
+
+class BetSettlement(BaseModel):
+    bet_uuid: str
+    outcome: bool
```

Comment:
where is this outcome determined

Reply:
You would be passing in a BetSettlment object when calling /settle which should have the uuid & outcome

Reply:
The outcome is determined manually for MVP?